include/Particles.h

```diff
@@ -113,6 +113,7 @@ class Particles : public ManagedAllocation {
     this->Z[indx] = Z;
     this->charge[indx] = charge;
     this->amu[indx] = amu;
+    /* Captain! This segfaults */
```

Comment:
I wonder if this is just when indx is outside the valid range, or if there is something else going on here.